### PR TITLE
Adds Purchase model

### PR DESCRIPTION
**Pull request by jondubin**
@Huntrr 
@webfreak7

### CODE REVIEW

File: app/models/purchase.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from .. import db
+
+
+class Purchase(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+
+    # model relationships
+    merchant_id = db.Column(db.Integer, db.ForeignKey('user.id'))
```
Comment (Huntrr) [inline app/models/purchase.py]: Isn't it `users.id`? I keep getting our pluralizations mixed up.

File: app/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from .. import db
+
+
+class Purchase(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```
Comment (Huntrr) [inline app/models/purchase.py:7]: Where is `__tablename__`?

File: app/models/purchase.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    # model relationships
+    merchant_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    listing_id = db.Column(db.Integer, db.ForeignKey('listing.id'))
+
+    # purchase properties
+    purchase_quantity = db.Column(db.Integer)
```
Comment (Huntrr) [inline app/models/purchase.py]: should this be `purchase_quantity` or `quantity`?

File: app/models/purchase.py (hunk 1 above)
Comment (jondubin) [inline app/models/purchase.py]: Fixed

File: app/models/purchase.py (hunk 2 above)
Comment (jondubin) [inline app/models/purchase.py:7]: Fixed

File: app/models/purchase.py (hunk 3 above)
Comment (jondubin) [inline app/models/purchase.py]: Fixed all of the fields

**Pull request merged**